_pages/icml_2020/icml-2020-apply-es.html

```diff
@@ -0,0 +1,10 @@
+---
+layout: main
+permalink: /icml-2019-apply-es
+title: LXAI @ ICML
+published: true
```

Comment:
Until information is updated on the Spanish version of this site, published should be false.